src/libs/Network/NetworkStore.ts

```diff
@@ -82,85 +78,51 @@ Onyx.connect({
             triggerReconnectCallback();
         }
 
-        offline = Boolean(network.shouldForceOffline) || network.isOffline;
+        offline = Boolean(network.shouldForceOffline) || !!network.isOffline;
     },
 });
 
-/**
- * @returns {Object}
- */
-function getCredentials() {
+function getCredentials(): Credentials | null {
     return credentials;
 }
 
-/**
- * @returns {Boolean}
- */
-function isOffline() {
+function isOffline(): boolean {
     return offline;
 }
 
-/**
- * @returns {String}
- */
-function getAuthToken() {
+function getAuthToken(): string | null {
     return authToken;
 }
 
-/**
- * @param {String} command
- * @returns {[String]}
- */
-function isSupportRequest(command) {
-    return _.contains(['OpenApp', 'ReconnectApp', 'OpenReport'], command);
+function isSupportRequest(command: string): boolean {
+    return ['OpenApp', 'ReconnectApp', 'OpenReport'].includes(command);
 }
 
-/**
- * @returns {String}
- */
-function getSupportAuthToken() {
+function getSupportAuthToken(): string | null {
     return supportAuthToken;
 }
 
-/**
- * @param {String} newSupportAuthToken
- */
-function setSupportAuthToken(newSupportAuthToken) {
+function setSupportAuthToken(newSupportAuthToken: string) {
     supportAuthToken = newSupportAuthToken;
 }
 
-/**
- * @param {String} newAuthToken
- */
-function setAuthToken(newAuthToken) {
+function setAuthToken(newAuthToken: string | null) {
     authToken = newAuthToken;
 }
 
-/**
- * @returns {String}
- */
-function getCurrentUserEmail() {
+function getCurrentUserEmail(): string | null {
```

Comment:
Hmmm should we make this always a string and have it be the empty string when it is not set?

Reply:
Same reasoning: https://github.com/Expensify/App/pull/28031#discussion_r1345601109